File: c#/EmployeeManagement/Models/FTAdmin.cs
using System;

namespace EmployeeManagement.Models
{
    public class FTAdmin : FTEmployee
    {
        public string AccessCode { get; }

        public FTAdmin(int id, string firstName, string lastName, DateOnly dateOfBirth, string position, Departments department, double salary, string accessCode)
            : base(id, firstName, lastName, dateOfBirth, position, department, salary)
        {
            AccessCode = !string.IsNullOrWhiteSpace(accessCode)
                ? accessCode
                : throw new ArgumentException("Access code cannot be null or empty.", nameof(accessCode));
        }

        public override void ReportToManager()
        {
            Console.WriteLine($"FT Admin {FirstName} {LastName} reports to the manager and uses access code {AccessCode}.");
        }

        public override string ToString()
        {
            return base.ToString() + $", AccessCode={AccessCode}";
        }
    }
}

File: Java/FTAdmin.java
public class FTAdmin extends FTEmployee {

    private String access_code;

    public FTAdmin(
        int id,
        String first_name,
        String last_name,
        String dob,
        String position,
        Departments department,
        double salary,
        String access_code
    ) {
        super(id, first_name, last_name, dob, position, department, salary);
        this.access_code = access_code;
    }

    @Override
    public void reportToManager() {
    }
}
